[2024-05-01T17:59:41.116+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.extract_data manual__2024-05-01T17:59:28.974851+00:00 [queued]>
[2024-05-01T17:59:41.136+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.extract_data manual__2024-05-01T17:59:28.974851+00:00 [queued]>
[2024-05-01T17:59:41.137+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 4
[2024-05-01T17:59:41.207+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): extract_data> on 2024-05-01 17:59:28.974851+00:00
[2024-05-01T17:59:41.212+0000] {standard_task_runner.py:60} INFO - Started process 114 to run task
[2024-05-01T17:59:41.216+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'main_dag', 'extract_data', 'manual__2024-05-01T17:59:28.974851+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp2n37x296']
[2024-05-01T17:59:41.219+0000] {standard_task_runner.py:88} INFO - Job 27: Subtask extract_data
[2024-05-01T17:59:41.317+0000] {task_command.py:423} INFO - Running <TaskInstance: main_dag.extract_data manual__2024-05-01T17:59:28.974851+00:00 [running]> on host 54fd76cc0661
[2024-05-01T17:59:41.451+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T17:59:28.974851+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-01T17:59:28.974851+00:00'
[2024-05-01T17:59:42.087+0000] {logging_mixin.py:188} INFO - An error occured: [Errno 2] No such file or directory: 'trip_data_query.sql'
[2024-05-01T17:59:42.088+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-05-01T17:59:42.105+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=extract_data, execution_date=20240501T175928, start_date=20240501T175941, end_date=20240501T175942
[2024-05-01T17:59:42.218+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-01T17:59:42.278+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
